tests/lifecycle/textanalytics_cases_test.go

```diff
@@ -0,0 +1,122 @@
+// +build !unit
+
+package lifecycle
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// Sentiment struct for credentials test
+type Sentiment struct {
+	Documents []Documents `json:"documents"`
+}
+
+// Documents struct for credentials test
+type Documents struct {
+	Language string `json:"language"`
```

Comment:
From the doc, is the `Language` useless? Also, it is a little confusing to me to pass the language when detecting language...

Reply:
I apologize, I added an incorrect doc. This test returns a sentiment score.

Here is the correct doc.

https://westus.dev.cognitive.microsoft.com/docs/services/TextAnalytics.V2.0/operations/56f30ceeeda5650db055a3c9